webviews/src/home.css

```
@@ -0,0 +1,208 @@
+  --home-bg: light-dark(#f6f7f9, #111214);
+  --home-fg: light-dark(#15171a, #f3f4f5);
+  --home-muted: light-dark(#626973, #a9b0ba);
+  --home-border: light-dark(rgb(0 0 0 / 12%), rgb(255 255 255 / 14%));
+  --home-card: light-dark(rgb(255 255 255 / 82%), rgb(255 255 255 / 7%));
+  --home-card-hover: light-dark(#fff, rgb(255 255 255 / 11%));
+  --home-shadow: light-dark(rgb(20 26 35 / 10%), rgb(0 0 0 / 20%));
+  --home-blue: #0a84ff;
+  --home-green: #25a244;
+  --home-violet: #9b5cff;
+  --home-orange: #ff9500;
+  background: transparent;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)